src/api/app/models/bs_request.rb

```diff
@@ -33,7 +33,7 @@ class SaveError < APIException
   scope :to_accept, -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
   # Scopes for collections
   scope :with_actions, -> { joins(:bs_request_actions).distinct.order(priority: :asc, id: :desc) }
-  scope :in_states, ->(states) { where('bs_requests.state in (?)', states).references(:bs_requests) }
+  scope :in_states, ->(states) { where('bs_requests.state in (?)', states) }
```

Comment:
I guess so. But I would say this is out of the scope of this PR. We can change this later on.